### PR TITLE
macOS Build/Packaging

### PR DESCRIPTION
**Pull request by james-t-bloom**
Ref: Log-13026

### CODE REVIEW

**Comment by stsantilena** [comment]
Maybe just some information in the description about the general PR structure, i.e. the general steps around packaging it up for brew?

File: Jenkinsfile [hunk 1]
```diff
@@ -72,6 +72,32 @@ pipeline {
...
+                    }
+                    steps {
+                        withCredentials([[
+                                            $class: 'AmazonWebServicesCredentialsBinding',
+                                            credentialsId: 'aws',
+                                            accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                                            secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
+                                            ]]){
+                            sh """
+                                 source $HOME/.cargo/env
```
Comment (dkhokhlov) [inline Jenkinsfile:95]: re:  rust  toolchain update
rustup is pretty quick. meaning - it is better to install it during the build. specific version that will be defined here. then no need to ask releng team to update it. installing it into specific folder under custom workspace will provide better isolation between branch builds against different rust versions.

File: docs/MACOS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Mezmo Agent on MACOS
```
Comment (dkhokhlov) [inline docs/MACOS.md:1]: macOS - to please mac funs? here and in other places below.

**Comment by james-t-bloom** [comment]
> 

that exists, i'll link it to you

File: docs/MACOS.md (hunk 2 above)
Comment (james-t-bloom) [reply docs/MACOS.md:1]: updated

**Review by stsantilena — APPROVED** [review]
@james-t-bloom if you just link the doc you created around this, that'd be good. Other than that, this LGTM.

**Pull request merged**